core/scripts/sql/texera_ddl.sql

```diff
@@ -84,6 +85,16 @@ CREATE TABLE IF NOT EXISTS workflow
 ) ENGINE = INNODB,
   AUTO_INCREMENT = 1;
 
+CREATE TABLE IF NOT EXISTS commentbox
```

Comment:
I don't think this will be tracked with the versioning since it is mapped to a single workflow not a version

Reply:
adding to that, I thought all the comment-related contents are part of `workflow`, per our discussion. Why do we have a separate commentbox table?